tests/bearlib/aspects/ExceptionsTest.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+from coalib.bearlib.aspects.exceptions import AspectLookupError
+
+
+class ExceptionTest:
+
+    def test_AspectLookupError(self):
+        with pytest.raises(AspectLookupError) as exc:
```

Comment:
do not use pytest specific functions in the unit tests.  We have an issue to remove them.